resources/js/Pages/Landing/SearchPanel.vue

```diff
@@ -418,7 +418,13 @@ function selectCountry(country) {
       </div>
     </div>
     <div class="mb-4 mt-2 w-full px-2 lg:px-3">
-      <p class="text-slate-500 ">
+      <p v-if="filteredCities.length == 1" class="text-slate-500 ">
+        {{ __('Found') }} {{ filteredCities.length }} {{ __('result') }}
+      </p>
+      <p v-else-if="((filteredCities.length % 10 == 2 || filteredCities.length % 10 == 3 || filteredCities.length % 10 == 4) && (filteredCities.length % 100 != 11 && filteredCities.length % 100 != 12 && filteredCities.length % 100 != 13 && filteredCities.length % 100 != 14))" class="text-slate-500 ">
```

Comment:
If you really want to show genitive form, I think we can simplify this. 
Instead of this all complicated logic, we can just write translation like that:

"Ilość wyników: <tu wstaw liczbę>"

and we don't have problem with genivite form anymore.


And this is beyond of this pr, but here "Ta aplikacja pomoże Ci z planowaniem Twoich wakacji, lub podróży biznesowych." it shouldn't be a comma.